presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -374,8 +373,15 @@ private RelationPlan createInsertPlan(Analysis analysis, Insert insertStatement)
                     Expression cast = noTruncationCast(input.toSymbolReference(), queryType, tableType);
                     assignments.put(output, cast);
                 }
+                columnsToBeInserted.add(columns.get(column.getName()));
+                visibleColumnsBuilder.add(column);
```

Comment:
do you need two collections? You are addings same set of columns to each of them.

Reply:
columnsToBeInserted has the ColumnHandle which we might need to pass for beginInsert operation so we might need it

Reply:
`columnsToBeInserted` can be easily constructed  from `visibleColumnsBuilder`.

`visibleColumnsBuilder.stream().map(column -> columns.get(column.getName()))).collect(...);`

Reply:
have applied